script/setup.js

```diff
@@ -63,9 +63,17 @@ try {
 
 	const { defaultOwner, defaultRepository } = await getDefaultSettings();
 
+	intro(
+		chalk.blue(
+			"Enter the details to hydrate your repository for your new package!"
+		)
+	);
+	console.log();
+
 	async function getPrefillOrPromptedValue(key, message, placeholder) {
 		if (values[key]) {
-			console.log(chalk.grey(`Pre-filling ${key} to ${values[key]}.`));
+			console.log(chalk.grey(`  Pre-filling ${key} to ${values[key]}.`));
```

Comment:
[Nit] I think this will make it line up a little better?

```suggestion
			console.log(chalk.grey(`│  Pre-filling repository to yippee.`));
```

It'd be nice to have something built into Clack... maybe something to file as a feature request?

Reply:
Right, this is a nice idea.